lib/charms/bind/v0/dns_record.py

```diff
@@ -623,14 +624,16 @@ def __init__(self, charm: ops.CharmBase, relation_name: str = DEFAULT_RELATION_N
 
     def get_remote_relation_data(
         self,
-    ) -> Optional[Tuple[DNSRecordRequirerData, DNSRecordProviderData]]:
-        """Retrieve the remote relation data.
+    ) -> List[Optional[Tuple[DNSRecordRequirerData, DNSRecordProviderData]]]:
```

Comment:
Doesn't 'List` instead of  `List[Optional]` suffice?

Reply:
You're right